code/tests/core/test_event.brs

```diff
@@ -69,4 +69,69 @@ sub TC_adb_ResponseEvent_empty_data()
     UTF_assertEqual({}, event.data)
     UTF_assertFalse(_adb_isEmptyOrInvalidString(event.uuid))
     UTF_assertFalse(_adb_isEmptyOrInvalidString(event.timestamp))
-end sub
\ No newline at end of file
+end sub
+
+' target: _adb_IdentityResponseEvent()
+' @Test
+sub TC_adb_IdentityResponseEvent()
```

Comment:
Some of these comparisons will become simpler if we add a test utility method to compare objects/JSON instances. Can you create a GH issue to track this?

Reply:
https://github.com/adobe/aepsdk-roku/issues/176